events/activityMonitor.ts

```diff
@@ -0,0 +1,23 @@
+import childProcess from 'child_process';
```

Comment:
Please use the modern way of importing, with `node:` prefix.

https://2ality.com/2021/12/node-protocol-imports.html